src/quicr_server.cpp

```diff
@@ -140,47 +130,23 @@ QuicRServer::subscriptionEnded(const quicr::Namespace& /* quicr_namespace */,
 
 void
 QuicRServer::sendNamedObject(const uint64_t& subscriber_id,
-                             const quicr::Name& quicr_name,
-                             [[maybe_unused]] uint8_t priority,
-                             [[maybe_unused]] uint64_t best_before,
                              [[maybe_unused]] bool use_reliable_transport,
-                             bytes&& data)
+                             const messages::PublishDatagram& datagram)
```

Comment:
Should this be messages::PublishDatagram&&?

Reply:
No, it's not a move. It's retained.

Reply:
We don't want to move the datagram? Not even to reduce copying the payload vector?